src/credentials_template.h

```diff
@@ -1,5 +1,15 @@
 #include <Preferences.h>
 
+
+/*  template file for preloading credentials instead of reading from the file system 
+ *  only applicable for [env:demo-unit-preloaded]
+ *
+ *  to use this file, copy this template to "credentials.h" and fill in the applicable values.  
+ *  the list below is non-exhaustive and additional options can be added as needed.
+ *  this will override anything in the file system on every startup.
+*/
```

Comment:
Should we add the template file to the .gitignore too? Add one more layer of protection requiring someone to supercede the ignore before accidentally checking in credentials in this file

Reply:
AFAIK if the file is in the repo, the .gitignore won't do quite what we want.